cmd/vc-rest/startcmd/start.go

```diff
@@ -432,11 +437,38 @@ func startEdgeService(parameters *vcRestParameters, srv server) error {
 		return err
 	}
 
-	handlers := vcService.GetOperations()
-	router := mux.NewRouter()
+	holderService, err := restholder.New(&holderops.Config{TLSConfig: &tls.Config{RootCAs: rootCAs},
+		StoreProvider: edgeServiceProvs.provider, KeyManager: localKMS, Crypto: crypto,
+		VDRI: vdri, Domain: parameters.blocDomain})
+	if err != nil {
+		return err
+	}
 
-	for _, handler := range handlers {
-		router.HandleFunc(handler.Path(), handler.Handle()).Methods(handler.Method())
+	switch parameters.mode {
+	case string(verifier), string(issuer):
+		handlers := vcService.GetOperations()
+
+		for _, handler := range handlers {
```

Comment:
This `for loop` inside each case looks common. May be we can extract it out of switch